geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/kmeans/mapreduce/KMeansMapReduce.java

```diff
@@ -76,59 +65,43 @@
 		private final BytesWritable outputValWritable = new BytesWritable();
 		private final GeoObjectDimensionValues association = new GeoObjectDimensionValues();
 		protected CentroidExtractor<Object> centroidExtractor;
-		protected AnalyticItemWrapperFactory<Object> itemWrapperFactory;		
+		protected AnalyticItemWrapperFactory<Object> itemWrapperFactory;
 
-		AssociationNotification<Object> centroidAssociationFn = new AssociationNotification<Object>() {
-			@Override
-			public void notify(
+		AssociationNotification<Object> centroidAssociationFn = new AssociationNotification<Object>()
+		{
+			@Override public void notify(
 					final CentroidPairing<Object> pairing ) {
 				outputKeyWritable.set(
-						pairing.getCentroid().getGroupID(),
-						pairing.getCentroid().getID());
+						pairing.getCentroid().getGroupID(), pairing.getCentroid().getID());
```

Comment:
I blame this on the eclipse formatter.  I downloaded luna, imported the format, and ran it on the file and it doesn't change anything.   If I make, say, pairing.getCentroid().getGroupID(), pairing.getCentroid().getID());  take two lines it doesn't change it back though either.